docs/weave/tutorial/11-cron.md

```diff
@@ -0,0 +1,325 @@
+---
+id: cron-tasks
+title: Scheduled Tasks
+sidebar_label: Scheduled Tasks
+---
+
+In almost every real time software, the need to schedule jobs to be executed at a certain time in
+future is constant. In blockchain context, this could be sending tokens, distributing revenue,
+scheduling a proposal, basically anything that could be executed as a transaction (a change of
+state). The early generation of blockchain protocols had a hard time assuring the execution time
+of a transaction precisely is impossible due to the indeterministic nature of the consensus algorithms.
+One of the wonders of Tendermint Consensus engine is the ability to roughly approximate the execution time of a
+future block thanks to **BFT Time feature**.
+[Tendermint BFT Time](https://github.com/tendermint/tendermint/blob/v0.32.7/docs/spec/consensus/bft-time.md) 
+document is a very technical and complicated specification, but long story short, execution time of a proposed block is
+get by calculating the median of latest block's commit voting times.
+
+[CRON](https://en.wikipedia.org/wiki/CRON) is no different than a regular message except the fact
+that CRON tasks are initiated by a ticker and processed by a different handler that is not
+accesible from the route. This means you cannot access the CRON handler from outside but you can
```

Comment:
This might need a better wording.

Some sentences before you write 

> Cron stack is a minimal router [...]

and here you write 

> [...] processed by a different handler that is not accesible from the route [...]
> [...] you can trigger a CRON task by a regular, routed handler.

I think the problem is in wording. 

Cron is not a router nor a handler. It is a functionality (mechanism?) that allows to schedule execution/handling of a message in the future. It can use any router instance in order to process scheduled messages. I don't know what exactly cron is in this context. I think it is important to figure this out and write at the beginning, using the same wording throughout this document.

I think it would be good to come up with a name for the functionality that is using cron and non-cron controlled routers.